FrenetOptimalTrajectory/fot.py

```diff
@@ -47,18 +49,20 @@ def main():
         "kt": 0.1,
         "ko": 0.1,
         "klat": 1.0,
-        "klon": 1.0
+        "klon": 1.0,
+        "num_threads": thread_num, # set 0 to avoid using threaded algorithm
     }
+    
     # static elements of planner
     wx = initial_conditions['wp'][:, 0]
     wy = initial_conditions['wp'][:, 1]
     obs = np.array(conds['obs'])
 
     # simulation config
-    show_animation = True
```

Comment:
Why was this line removed?

Reply:
I made `show_animation` part of the function parameters to call `run_fot`, thus no need to declare it here.